perses/app/setup_relative_calculation.py

```diff
@@ -298,8 +299,7 @@ def getSetupOptions(filename):
         _logger.info(f"\t'softcore_v2' not specified: default to 'False'")
 
     _logger.info(f"\tCreating '{trajectory_directory}'...")
-    assert (not os.path.exists(trajectory_directory)), f'Output trajectory directory "{trajectory_directory}" already exists. Refusing to overwrite'
-    os.makedirs(trajectory_directory)
+    os.makedirs(trajectory_directory, exist_ok=True)
```

Comment:
What happens if the directory already exists? Do contents get overwritten? If that's the case I think maybe we shouldn't do that or at least we should warn the users. What do you think?

Reply:
If it exists, it does not get overwritten. That function acts like the UNIX utility `mkdir` where if the directory already exists, you get an error ie:
```
$ mkdir foo
$ mkdir foo
mkdir: cannot create directory ‘foo’: File exists
```

Reply:
Yeah, I agree. So, I don't know if I'm missing something else, but when the directory exists there wouldn't be an error and the contents in it then could get overwritten?

Reply:
It won't overwrite the contents:
```
$ ls foo
bar
$ python
Python 3.8.5 (default, Sep  4 2020, 07:30:14) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> os.makedirs("foo", exist_ok=True)
>>> 
$ ls foo/
bar
```
Also the docs don't mention that the contents will be overwritten 
https://docs.python.org/3/library/os.html#os.makedirs

Reply:
I will be making a new issue to discuss about this a bit further. What if the user uses the same directory name without realizing it already exists?